### DISCUSSION TITLE
show loading Indicator when Tiles are loading

### DISCUSSION THREAD

**Discussion by dario-digregorio** [fleaflet/flutter_map#1557] [Q&A]
So basically I would like to show a loading indicator when Tiles are loading. Which way would be the easiest solution. I tried to use a Interceptor for HttpClient and then count up/down a int. But this seems a bit hacky.

Could there be a easy way?

**Comment by ibrierley** [comment]
I'm not sure there's a simple intuitive way. I'd probably question the need though...you will just be indicating what is already apparent, i.e a missing tile (for which there may be a backup lower zoom tile in place)? How long are your tiles taking to load (it may be worth looking at a source of any issue) ?
